src/lightning_app/cli/lightning_cli_create.py

```diff
@@ -75,5 +75,5 @@ def create_cluster(
         external_id=external_id,
         edit_before_creation=edit_before_creation,
         cost_savings=not enable_performance,
-        wait=wait,
+        wait=not no_wait,
```

Comment:
TBH, I'd rather don't use the flag negation - adds additional entanglement, and switch to `no_wait` in `cluster_manager.create` and `cluster_manager.delete` as well.

Reply:
Yeah, call this then `--async` since it's more positive formulation

Reply:
Thanks for the feedback!